_1327/emails/templates/emails_view.html

```diff
@@ -0,0 +1,55 @@
+{% extends "base_with_sidebar.html" %}
+{% load i18n %}
+{% load static %}
+{% load compile_static %}
+{% load mptt_tags %}
+
+{% block css %}
+    {{ block.super }}
+    <link rel="stylesheet" href="{% static 'less/emails.less'|compile %}" />
+{% endblock %}
+
+{% block title %}{{ email.subject_nonempty }}{% endblock title %}
+
```

Comment:
breadcrumbs would be super nice. e.g., "Email archive / 2019 / 01 / Title of the email I'm looking at"
same for search etc.